.github/workflows/autograding.yml

```diff
@@ -25,37 +23,27 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - name: Student Repo
         uses: actions/checkout@v3
-        timeout-minutes: 10
```

Comment:
Add a 10 min limit to the build. 
Reduce step to 2 mins. 
They will still be at least 2x average.